### ISSUE TITLE
Object path##### has an empty 'd' attribute. Please delete this object from your document.

### ISSUE THREAD

**Issue by AJuliette** [inkstitch/inkstitch#2817]
I couldn't get a "Simulator / Realistic preview" (it loaded for hours) and when I closed Inkstitch I got this message: 

<img width="471" alt="Screenshot 2024-04-01 at 21 46 10" src="https://github.com/inkstitch/inkstitch/assets/42354334/378cdd83-bd2e-4dfd-a9a0-80d2663a1267">

Here is my file:
[
![coast-map](https://github.com/inkstitch/inkstitch/assets/42354334/8ef395a6-3d19-4fe5-8f0a-236572304b24)
](url)

**Comment by tyrosinase** [comment]
It's not really a stitchable file yet: it's got thousands of objects (layered, which is fine for print or screen but doesn't make sense for stitching) and is almost a foot across. That's an absolutely enormous amount of calculation for the previewer, and you probably don't have enough memory for it.

You will want to start with a more modest project until you get a feel for how machine embroidery works and what the thread is capable of.

**Comment by AJuliette** [comment]
Hi @tyrosinase, thanks for taking the time to review my PR and my attached document.
I did not appreciate that you assumed I had no experience in machine embroidering, I did several projects that worked fine. They weren't as big as this one.
I don't think the error message is clear in this case, because that made me thought there was something going wrong in my design, not that it was too big. I'll look at a way to contribute and maybe improve the error message in this case, because I think it was misleading.

**Comment by kaalleen** [comment]
Over the time we tried to reduce error messages when we saw it was possible. So I hope Ink/Stitch will render most designs without error messages. Your file helped us to remove just an other one.

When we create embroidery designs we will need to categorize design parts as fill, satin and stroke or think of creative path ways to generate your specific look, reduce jump stitches as much as possible, take care of the pull compensation, etc. So even with a lot of experience every new design is a challenge and a creative journey and with every new file there are things to learn.

We see a lot of files from users who are just starting with their embroidery journey. When we see fill stitches on small shapes, no stitch routing to reduce jumps, no pull compensation, etc. we would try to give advice on how to create better embroidery. This can of course happen, when you send a file for a bug report in an unfinished state. I really appreciate that @tyrosinase is answering on issues and helps beginners to get started. Sorry if you already know more about digitizing and don't need the advice. I think though, that we rather post more info at the wrong place than missing out to help people who need it.

**Comment by AJuliette** [comment]
Thanks for taking the time to answer me in such details. For more context I go to a FabLab which uses PE-Design, I only use Ink/Stitch to see a rough draft of the embroidery from my computer before going to the FabLab to use their software. I had no problem before in the preview with Ink/Stitch like the one I stumbled into, I did not mean to throw a file at you without having done my research, but I understand that as my use of the software is pretty basic it was misleading.
